### PR TITLE
Readme changes

### PR DESCRIPTION
**Pull request by Nattalli**

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -12,36 +12,29 @@
...
 
 
-@pytest.mark.parametrize(
-    "attribute,attr_type",
-    [
-        ("name", str),
-        ("number", int)
-    ]
-)
+@pytest.mark.parametrize("attribute,attr_type", [("name", str), ("number", int)])
```
Comment (Abnormaltype) [inline tests/test_main.py:32]:
```suggestion
@pytest.mark.parametrize(
    "attribute,attr_type",
    [
        ("name", str),
        ("number", int)
    ]
)
```

File: tests/test_main.py [hunk 2]
```diff
@@ -12,36 +12,29 @@
...
-        "name",
-        "number"
-    ]
-)
+def remove_created_test_file(path):
+    if os.path.exists(path):
+        os.remove(path)
+
+
+@pytest.mark.parametrize("attribute", ["name", "number"])
```
Comment (Abnormaltype) [inline tests/test_main.py:24]:
```suggestion
@pytest.mark.parametrize(
    "attribute",
    [
        "name",
        "number"
    ]
)
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
more readable and addfixtures

File: tests/test_main.py [hunk 3]
```diff
@@ -12,22 +12,57 @@
...
 
 
+@pytest.fixture
+def specialty_init():
+    return Specialty("Math and Physic", 1)
+
+
+@pytest.fixture
+def student_init():
+    return Student("Ivan", "Ivanov", date.today(), 87.5, True, "+380999999999", "Kyiv")
```
Comment (danylott) [inline tests/test_main.py:26]: I believe this fixture should use `basic_student` function

**Review by danylott — APPROVED** [review]
LGTM

File: tests/test_main.py (hunk 3 above)
Comment (Nattalli) [reply tests/test_main.py:26]: Oh, yes, should

**Review by Abnormaltype — APPROVED** [review]
LGTM

**Pull request merged**